feature-libs/quote/components/details/edit/quote-details-edit.component.ts

```diff
@@ -0,0 +1,121 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { UntypedFormControl, UntypedFormGroup } from '@angular/forms';
+import { ICON_TYPE } from '@spartacus/storefront';
+
+export interface EditEvent {
+  editMode: boolean;
+  name?: string;
+  description?: string;
+  expirationTime?: Date;
+}
+
+export interface EditCard {
+  title: string;
+  paragraphs: Array<{
+    title: string;
+    text: string;
+    isTextArea?: boolean;
+    charactersLimit?: number;
+  }>;
+}
+
+@Component({
+  selector: 'cx-quote-details-edit',
+  templateUrl: './quote-details-edit.component.html',
+})
+export class QuoteDetailsEditComponent implements OnInit {
+  iconTypes = ICON_TYPE;
+  editForm: UntypedFormGroup;
+
+  @Output()
+  editCard: EventEmitter<EditEvent> = new EventEmitter();
+  @Output()
+  cancelCard: EventEmitter<boolean> = new EventEmitter();
+
+  @Input()
+  content: EditCard | null;
+
+  /**
+   * Cancels the view of the edit card tile
+   * by throwing the edit event with the edit mode set to 'false'.
+   */
+  cancel(): void {
+    this.cancelCard.emit(false);
+  }
+
+  /**
+   * Edits the card tile by throwing the edit event
+   * with the edit mode set to 'false' and the edited data.
+   */
+  edit(): void {
+    const event: EditEvent = {
+      editMode: false,
+    };
```

Comment:
or do we ever emit with editMode: true
```suggestion
    const event: SaveEvent = {
    };
```
and as already mentioned, maybe we should rename this to save event.

Reply:
We could discuss it.

Reply:
Done

Reply:
seems like you forgot to commit/save this change?
can you please rename the event to SaveEvent and remove the editMode property. 

@Uli-Tiger, done

Reply:
@Uli-Tiger, I have renamed the event but I would prefer to stick to emitting `editMode`